app/models/manageiq/providers/cisco_intersight/physical_infra_manager.rb

```diff
@@ -25,7 +25,7 @@ def self.description
     end
 
     def self.catalog_types
-      {"cisco_intersight" => N_("Cisco intersight")}
+      {"cisco_intersight" => N_("Cisco Intersight")}
```

Comment:
Total aside, I don't understand why this String is translatable, since it's a product name technically?  It makes sense for catalog items like "Generic", but not here.  I noticed other providers have it too, so I'm leaving it for now.

@Frank-NB Do you have any thoughts here?